notificationClient/notificationClient.go

```diff
@@ -0,0 +1,38 @@
+package notificationClient
+
+import (
+	"context"
+	"net/http"
+
+	e3dbClients "github.com/tozny/e3db-clients-go"
+	"github.com/tozny/e3db-clients-go/authClient"
+)
+
+type ToznyNotificationClient struct {
+	Host       string
+	authClient authClient.E3dbAuthClient
+}
+
+const (
+	// NotificationServiceBasePath is the base path used for notification-service calls
+	NotificationServiceBasePath = "v1/notification"
+)
+
+// CreateNotification pushes a notification via an internal endpoint. Requires the bootstrap client.
+func (nc *ToznyNotificationClient) CreateNotification(ctx context.Context, params CreateNotificationRequest) (*NotificationMeta, error) {
+	var result *NotificationMeta
+	path := nc.Host + "/internal/" + NotificationServiceBasePath + "/"
+	request, err := e3dbClients.CreateRequest(http.MethodPost, path, params)
+	if err != nil {
+		return result, e3dbClients.NewError(err.Error(), path, 0)
+	}
+	err = e3dbClients.MakeE3DBServiceCall(&nc.authClient, ctx, request, &result)
```

Comment:
In line with trying to push the greatest that is tsv1 , we should probably make this client authenticate it's calls with signature based auth (see storage v2/notes and identity clients for examples), it shouldn't be a change for the notification service itself as with either valid authentication method cyclops will construct a X-TOZNY-AUTHN header for the service to use as an auth check. (no need to hold off on dev deployments based on this change though)

Reply:
Oops. Didn't see the "no need to hold off" part. Oh well. Added that authenticator to notification-service@working